libafl_sugar/src/forkserver.rs

```diff
@@ -177,17 +173,33 @@ impl<'a, const MAP_SIZE: usize> ForkserverBytesCoverageSugar<'a, MAP_SIZE> {
                 ForkserverExecutorBuilder::new()
                     .program(self.program.clone())
                     .parse_afl_cmdline(self.arguments)
+                    .is_deferred_frksrv(true)
```

Comment:
@tokatoka pointed out that this won't work for _some_ targets, so it's better to not set it (afl++ solves this by scanning the binary, but we don't want to do that imho)